src/index.html

```diff
@@ -5,9 +5,198 @@
+        <ul class="nav__list">
+          <li class="nav__item nav--active">
+            <a href="#">HOME</a>
+          </li>
+          <li class="nav__item">
+            <a href="#">ABOUT US</a>
+          </li>
+          <li class="nav__item">
+            <a href="#">GALLERY</a>
+          </li>
+          <li class="nav__item">
+            <a href="#">CONSILTATION</a>
+          </li>
+          <li class="nav__item">
+            <a href="#">TESTIMONIALS</a>
+          </li>
+          <li class="nav__item">
+            <a href="#">CONTACTS</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+  </header>
+
+  <section class="body-section aboutUs">
+    <h1 class="black aboutUs__title">Miami best Real estate <a href="#" class="blue">here</a></h1>
+    <div class="aboutUs__text">
+      <a href="#" class="blue h2-title-link">Over 50 years experience in love.</a>
+
+      <div class="aboutUs__text-passag">
+        <p>        Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+        </p>
+      </div>
+      <div class="aboutUs__text-passage">
+        <p> The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.
+        </p>
+      </div>
+        <strong>We will never be late!</strong> <br>
+        <strong>Guaranteed secure transaction - 100%</strong>
+
+    </div>
+  </section>
+
+  <section class=" body-section gallery">
+    <h1 class="black h1__title">Gallery</h1>
+
+    <div class="row">
+      <div class="row__block row__block--2-1">
+        <img src="images/photos/1.jpg" alt="guardian box" class="row__item">
+        <img src="images/photos/2.jpg" alt="guardian box" class="row__item">
+      </div>
+      <div class="row__block-description">
+        <a href="#" class="blue row__block-description-link">Marina Palms</a> <span> / North Miami Beach, FL 33162</span>
+      </div>
+    </div>
+    <div class="row">
+      <div class="row__block row__block--1-1">
+        <img src="images/photos/3.jpg" alt="guardian box" class="row__item">
+        <img src="images/photos/4.jpg" alt="guardian box" class="row__item">
+      </div>
+      <div class="row__block-description">
+        <a href="#" class="blue row__block-description-link">Edition Residens</a> <span> / Miami Beach, FL 33139</span>
+      </div>
+    </div>
+    <div class="row">
+      <div class="row__block row__block--1-2">
+        <img src="images/photos/5.jpg" alt="guardian box" class="row__item">
+        <img src="images/photos/6.jpg" alt="guardian box" class="row__item">
+      </div>
+      <div class="row__block-description">
```

Comment:
```suggestion
      <div class="row__description">
```
на мой взгляд слово блок тут лишнее, старайся делать простые названия без явных слов